openassessment/templates/openassessmentblock/edit/oa_edit.html

```diff
@@ -18,13 +18,13 @@ <h6 id="oa_editor_window_title" class="title modal_window_title" >{% trans "Open
         </div>
 
         <div id="openassessment_validation_alert" class="covered">
-            <i class="openassessment_alert_icon"></i>
+            <span class="openassessment_alert_icon"></span>
```

Comment:
Yeah it should, forgot to add it in. The rest of the icons in the templates look like they have it so I'll just fix this one.